config/settings.yml

```diff
@@ -97,6 +97,7 @@
     :purge_window_size: 10000
 :ems:
   :ems_redhat:
+    :resolve_ip_addresses: true
```

Comment:
I think this setting is not really self explanatory. But I dont know how could be really short and good.

`resolve_ip_address_for_ovirt_manager_connect: true` ?